addons/core/addon/components/dropdown/index.hbs

```diff
@@ -3,7 +3,7 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<Hds::Dropdown @onClose={{this.close}} as |DD|>
+<Hds::Dropdown @height='314px' @onClose={{this.close}} as |DD|>
```

Comment:
Why 314 specifically out of curiosity lol

Reply:
That is the height we have in the wireframes. This can change if we need it to but I felt this was a good height based on the designs.